src/general/OptionsMenu.cs

```diff
@@ -303,4 +358,9 @@ private void OnCloudResolutionSelected(int index)
     {
         Settings.CloudResolution = CloudIndexToResolution(index);
     }
+
+    private void OnMSAAResolutionSelected(int index)
+    {
+        Settings.MSAAResolution = MSAAIndexToResolution(index);
```

Comment:
This should instantly apply this like the sound and window fullscreen options do.
And there should be a specific method in the settings class for applying it. Right now this doesn't even get applied when restarting the game.